src/KPM.jl

```diff
@@ -164,24 +168,25 @@ function iterateKPM!(ket0::A, ket1::A, adjh::Adjoint, (center, halfwidth)) where
     nzv = nonzeros(h)
     rv = rowvals(h)
     T = eltype(S)
-    μ = μ´ = zero(T)
+    #μ = μ´ = zero(T)
+    μ = μ´ = zeros(T,Threads.nthreads())
```

Comment:
Mmm, this is dangerous. Since `zeros` returns an array, this line makes `μ` and `μ´` point to the *same* array. I suspect this is not what you intended.

Reply:
Yes, you are right! Let me fix it